webapp/channels/src/i18n/en.json

```diff
@@ -4061,12 +4061,12 @@
-  "leave_team_modal_private.desc": "**You will be removed from {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels} } on this team.** If the team is private you won't be able to rejoin it the team without an invitation from another team member. Are you sure?",
-  "leave_team_modal_public.desc": "**You will be removed from {num_of_public_channels} public { num_of_public_channels,plural,one {channel} other {channels} } on this team.** Are you sure?",
-  "leave_team_modal.desc": "**You will be removed from {num_of_public_channels} public { num_of_public_channels,plural,one {channel} other {channels}} and {num_of_private_channels} private { num_of_private_channels,plural,one {channel} other {channels}} on this team.** If the team is private you won't be able to rejoin it without an invitation from another team member. Are you sure?",
+  "leave_team_modal_guest_only_private.description": "<strong>You will be removed from {num_of_private_channels}</strong> private {num_of_private_channels,plural,one {channel} other {channels}} on this team. You won't be able to rejoin it without an invitation from another team member. Are you sure?",
+  "leave_team_modal_guest_only_public.description": "<strong>You will be removed from {num_of_public_channels}</strong> public {num_of_public_channels,plural,one {channel} other {channels}} on this team. You won't be able to rejoin it without an invitation from another team member. Are you sure?",
+  "leave_team_modal_guest.description": "<strong>You will be removed from {num_of_public_channels}</strong> public {num_of_public_channels,plural,one {channel} other {channels}} and <strong>{num_of_private_channels}</strong> private {num_of_private_channels,plural,one {channel} other {channels}} on this team. You won't be able to rejoin it without an invitation from another team member. Are you sure?",
+  "leave_team_modal_private.description": "<strong>You will be removed from {num_of_private_channels}</strong> private {num_of_private_channels,plural,one {channel} other {channels}} on this team. If the team is private you won't be able to rejoin it without an invitation from another team member. Are you sure?",
+  "leave_team_modal_public.description": "<strong>You will be removed from {num_of_public_channels}</strong> public {num_of_public_channels,plural,one {channel} other {channels}} on this team. Are you sure?",
+  "leave_team_modal.description": "<strong>You will be removed from {num_of_public_channels}</strong> public {num_of_public_channels,plural,one {channel} other {channels}} and <strong>{num_of_private_channels}</strong> private {num_of_private_channels,plural,one {channel} other {channels}} on this team. If the team is private you won't be able to rejoin it without an invitation from another team member. Are you sure?",
```

Comment:
These didnt got updated with the changes you made can you cross verify

Reply:
They got updated, I guess. The initial suffix was `desc` then I changed it to `description`

Reply:
Check the description